packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-mobile.tsx

```diff
@@ -132,12 +183,15 @@ const Numbers = observer(
                     }}
                 />
             );
-            if (parseInt(value) < min || parseInt(selected_duration) > max) {
+            if (
+                parseInt(value as unknown as string) < Number(min) ||
+                parseInt(selected_duration as unknown as string) > Number(max)
```

Comment:
same here

Reply:
I removed `as unknown` for `selected_duration`. 
But `selected_duration` is number (checked in console), so `parseInt()` is not a proper method. That is why I replaced it with `Math.trunc()` as it do the same thing with numbers